src/XMLimport.cpp

```diff
@@ -957,6 +957,13 @@ void XMLimport::readHost(Host* pHost)
         pHost->setLargeAreaExitArrows(false);
     }
 
+    if (attributes().hasAttribute(qsl("BoldIsBright"))) {
```

Comment:
When this merges, there was some work done in PR #7036 to use code like this you may want to consider: `setBoolAttributeWithDefault(qsl("BoldIsBright"), pHost->mBoldIsBright, true);`.  It won't show as a conflict, I think.

Reply:
As that has been merged I may get something to do when I try to click on "Update branch". :grinning: